setup.py

```diff
@@ -29,20 +29,16 @@
 # 'Development Status :: 5 - Production/Stable'
 release_status = "Development Status :: 5 - Production/Stable"
 dependencies = [
-    "grpcio >= 1.47.0, < 2.0dev",  # https://github.com/googleapis/python-bigquery/issues/1262
-    "grpcio >= 1.49.1, < 2.0dev; python_version>='3.11'",
     # NOTE: Maintainers, please do not require google-api-core>=2.x.x
     # Until this issue is closed
     # https://github.com/googleapis/google-cloud-python/issues/10566
-    "google-api-core[grpc] >= 1.31.5, <3.0.0dev,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0",
-    "proto-plus >= 1.15.0, <2.0.0dev",
+    "google-api-core >= 1.31.5, <3.0.0dev,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0",
```

Comment:
Is there a reason we are not referencing the `[grpc]` install of `google-api-core`?

Reply:
Per my conversation with Tim Swast, there was no longer a reason to since it is now an optional dependency? Maybe he can answer this question better @tswast

Reply:
`grpc` will pull in protobuf and grpc libraries. I don't think we actually need those for the core REST API stuff.